server/views/add-articles.hbs

```diff
@@ -2,14 +2,21 @@
     <h4>Title</h4>
     <input type="title" name="title" />
     <br/>
+    <h4> Categories:</h4>
+        <select>
+            <option value="null" selected>Choose a Category</option>
+            {{#each category}}
+            <option value={{ this.title }} name="categoryTitle">{{ this.title }}</option>
```

Comment:
the value should be this._id I would think - you display to the user the title, but you want to save the id in the database

Reply:
great that is true